lib/beaker/platform.rb

```diff
@@ -24,10 +24,11 @@ class Platform < String
                      "precise" => "1204",
                      "lucid"   => "1004",
                    },
-        :osx =>    { "sierra"    => "1012",
-                     "elcapitan" => "1011",
-                     "yosemite"  => "1010",
-                     "mavericks" => "109",
+        :osx =>    { "highsierra" => "1013",
```

Comment:
I'm not sure why github shows this as a diff, but https://github.com/puppetlabs/beaker/pull/1468 may have previously added the highsierra codename?